docker/dockerfiles/Dockerfile

```diff
@@ -83,16 +83,18 @@ RUN export CHROMEDRIVER_RELEASE=$(curl http://chromedriver.storage.googleapis.co
       && sudo chmod +x /usr/local/bin/chromedriver
 
 # install mysql
-RUN curl -sSL -o /tmp/mysql-apt-config_0.8.9-1_all.deb https://dev.mysql.com/get/mysql-apt-config_0.8.9-1_all.deb \
+RUN curl -sSL -o /tmp/mysql-apt-config_0.8.12-1_all.deb https://dev.mysql.com/get/mysql-apt-config_0.8.12-1_all.deb \
   && echo "mysql-apt-config mysql-apt-config/select-server select mysql-5.7" | /usr/bin/debconf-set-selections \
-  && DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/mysql-apt-config_0.8.9-1_all.deb || apt-get -fy install \
-  && rm -rf /tmp/mysql-apt-config_0.8.9-1_all.deb \
+  && DEBIAN_FRONTEND=noninteractive dpkg -i /tmp/mysql-apt-config_0.8.12-1_all.deb || apt-get -fy install \
+  && rm -rf /tmp/mysql-apt-config_0.8.12-1_all.deb \
   && apt-get update \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     mysql-server \
     libmysqlclient-dev
-RUN service mysql start \
+RUN chown -R mysql:mysql /var/lib/mysql /var/run/mysqld \
+  && service mysql start \
   && echo "ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '';" | mysql \
+  && chown -R mysql:mysql /var/lib/mysql /var/run/mysqld \
```

Comment:
Nitpick: I don't think you need to run the same `chown` command twice since the changes to the directories will persist from the first time.

Reply:
I tried without first, but I was getting a hang on shutdown. Any idea why that would be?

Reply:
🤷‍♀yea I don't know then.